src/udp_wrap.cc

```diff
@@ -175,6 +175,20 @@ void UDPWrap::GetFD(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(fd);
 }
 
+int uv_sockaddr_for_family(int address_family,
+                           const char* address,
+                           const unsigned short port,
+                           struct sockaddr_storage* addr) {
+  switch (address_family) {
+  case AF_INET:
+    return uv_ip4_addr(address, port, reinterpret_cast<sockaddr_in*>(addr));
+  case AF_INET6:
+    return uv_ip6_addr(address, port, reinterpret_cast<sockaddr_in6*>(addr));
+  default:
+    CHECK(0 && "unexpected address family");
+    ABORT();
```

Comment:
I think the extra `ABORT();` can be dropped as unreachable code :)

Reply:
Sounds good, I'll remove it. I'll just let CI complete first. Thanks